pootle/static/js/editor/app.js

```diff
@@ -313,7 +313,8 @@ PTL.editor = {
     const hotkeys = mousetrap(document.body);
```

Comment:
im aware that its not directly the purpose of this PR - but should this be `moustrap("#editor")`